index.html

```diff
@@ -2,42 +2,46 @@
 <html lang="en">
 
 <head>
+    <title>Product Landing Page</title>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css" />
+    <link href="https://fonts.googleapis.com/css?family=Inter&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="styles.css">
-    <title>Product Landing Page</title>
 </head>
 
 <body>
-    <header id="header">
-        <nav id="nav-bar" class="nav-bar">
-            <img id="header-img" class="header-logo"
-                src="https://logosmarcas.net/wp-content/uploads/2020/12/Dota-2-Logo.png" width="60px" />
-            <ul class="ul">
-                <a class="nav-link active" href="#about">About</a>
-                <a class="nav-link" href="#download">Download</a>
-                <a class="nav-link" href="#play">Play</a>
+<header>
+    <nav class="toolbar">
+        <a class="header-logo" href="#"><img src="https://logosmarcas.net/wp-content/uploads/2020/12/Dota-2-Logo.png"
```

Comment:
Using an image as the header logo is fine, but it's important to make sure that the image has an appropriate alt attribute to provide context to users who may not be able to see the image. In this case, the image doesn't seem to have an alt attribute, which could be improved by adding a descriptive text alternative.